src/status_im/signing/core.cljs

```diff
@@ -272,14 +272,13 @@
                                           :success-event :signing/update-estimated-gas-success
                                           :error-event :signing/update-estimated-gas-error}})
        (fn [cofx]
-         (when-not (or maxFeePerGas gasPrice)
```

Comment:
We can remove `maxFeePerGas, gasPrice` keys on line 227, as they are no longer being used. Still curious how come lint didn't complain about unused variables 🤔